marimo/_cli/convert/markdown.py

```diff
@@ -232,11 +222,16 @@ def _tree_to_app(root: Element) -> str:
         )
         sources.append(get_source_from_tag(child))
 
+    header = root.get("header", None)
+    sandbox = root.get("sandbox", None)
```

Comment:
`pyproject` or `script_metadata` might be a better name, since you can include config in the sandbox as well (that get used outside the sandbox)

Reply:
Agreed